db/migrate/20220620170036_create_accounts.rb

```diff
@@ -0,0 +1,43 @@
+class CreateAccounts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :accounts do |t|
+      t.belongs_to :profile, polymorphic: true, null: false
+
+      t.timestamps
+    end
+
+    create_table :account_memberships do |t|
+      t.belongs_to :user, null: false, foreign_key: true
+      t.belongs_to :account, null: false, foreign_key: true
+
+      t.string :role, default: "admin", index: true
+
+      t.timestamps
+    end
+
+    # TODO: Write a test to ensure the correct account is migrated.
+    [Business, Developer].each do |model|
```

Comment:
I'm confused by this migration. It looks like this code is iterating through every business & developer record (in batches) and creating an account for each (along with an associated profile details). But then on lines 21 & 32 only 'business' is used as the variable name. The record that actually gets yielded to the block could be either a business or a developer, right? I realize the variable `|business|` could be called anything under the sun & it wouldn't make a functional difference, but from a readability standpoint it feels confusing to have Business and Developer called out on line 19 but then only reference businesses later in the block.

Reply:
Yes, you're right. We extracted the `[Business, Developer]` loop after writing this initially for businesses, and then ran out of time and stopped pairing before we could rename the variable.